### ISSUE TITLE
Offer a dual-license (MIT) or re-license

### ISSUE THREAD

**Issue by tofumatt** [localForage/localForage#892]
For projects licensed under the GPL 2.0 license (good examples being [WordPress.com](https://github.com/Automattic/wp-calypso/issues/28719) or [Gutenberg](https://github.com/WordPress/gutenberg)), our Apache 2.0 license is incompatible. See: https://github.com/Automattic/wp-calypso/issues/28719 for an example of this.

I'm fine either:

* re-licensing under an MIT license
* dual-licensing under Apache 2.0 and MIT

My preference is to change the license (in this case we'd be going even more permissive) to make things simpler. But I'm opening this issue to see if folks have any reasons why we should dual license on the next version. The aim of using an Apache 2.0 license is summed up here: https://opensource.com/article/18/2/how-make-sense-apache-2-patent-license, but I'm okay with MIT too.

Basically: would people be okay moving to an MIT License for the next minor release of localForage?

**Comment by thgreasi** [comment]
Cross posting for convenience & reference
> @tofumatt I have no objection on something like this but also don't have enough background. Imo dual licensing sounds like a smaller change but again I'm not an expert on licensing.

**Comment by tofumatt** [comment]
From the discussion in https://github.com/Automattic/wp-calypso/issues/28719 it seems like the WP.com team may not be interested in continued use of localForage, so I don't think it's worth us re-licensing just to suit them.

I think it's best to just close this and let Automattic sort out their licensing issues.
